client/src/main/java/model/FleetCreator.java

```diff
@@ -0,0 +1,59 @@
+package model;
+
+import gui.fields.Mast;
+
+import java.util.ArrayList;
+import java.util.List;
+/**
+ * It holds information regarding a player's fleet.
+ *
+ * @author Patrycja Mikulska
+ * @version 1.5
+ */
+
+public class FleetCreator {
+
+  private List<Position> positions;
```

Comment:
positions are updated but never queried. Is that ok?

Reply:
I think i removed it before, but this magic git sometimes messes with our code ;)